### DISCUSSION TITLE
Hurray, Masa CMS is here!

### DISCUSSION THREAD

**Discussion by guustnieuwenhuis** [MasaCMS/MasaCMS#6] [Announcements]
# Hurray, Masa CMS is here!

With great pride we're announcing the first release of Masa CMS!

This release is backwards compatible with Mura CMS and its main focus is on rebranding to Masa CMS. But, already there are two new features:

### Replaced CKFinder

Masa CMS does not come with a license for CKFinder, therefor the image upload functionality in the rich text editors (CKEditor) has been replaced with a version that does not rely on CKFinder.

If you want to add an image to your content, just drag-and-drop it into your content, Masa CMS takes care of the rest.

If you want to keep using CKFinder, all you need to do is bring your own license and configure it as described [here](https://github.com/MasaCMS/MasaCMS/wiki/Configure-CKFinder-license).

### Enable/Disable File Manager

Since the File Manager in Masa CMS depends heavily on CKFinder, it has now been disabled by default. You can enable it by updating the `filemanagerenabled` setting in `config/settings.ini.cfm` to `true`.

## Migrating from Mura CMS?

If you're migrating from Mura CMS, just follow [these instructions](https://github.com/MasaCMS/MasaCMS/wiki/Migrating-from-Mura-CMS-to-Masa-CMS).

***

For the release notes, see: https://github.com/MasaCMS/MasaCMS/releases/tag/7.2.0

**Comment by Tigerlady** [comment]
Just saw a tweet from Brad Wood about this.  I am so happy that someone has picked up the reigns for the CMS.

**Comment by Spiderwoman** [comment]
Sweet, thanks for doing this! This now gives hope for us still using the one-we-shall-not-name.

**Comment by brucebailey** [comment]
Trying to drag and drop an image on the front end and I am getting the following error:

layoutmanager.js:591 Uncaught ReferenceError: dragEl is not defined
at HTMLDivElement.initRegion_drop (layoutmanager.js:591)
at HTMLDivElement. (domselection.js:290)
initRegion_drop @ layoutmanager.js:591
(anonymous) @ domselection.js:290

I get the green + sign so it seems like it wants to go, but sadly, no

Any thoughts?

**Reply by guustnieuwenhuis** [reply]
Hi Bruce, that seems like something we'll need to investigate. Could you create an issue for this?
